subiquity/models/subiquity.py

```diff
@@ -93,6 +93,14 @@ def setup_yaml():
 ff02::2 ip6-allrouters
 """
 
+CLOUDINIT_CLEAN_FILE_TMPL = """\
+#!/bin/sh
+# Remove live-installer config artifacts when running: sudo cloud-init clean
```

Comment:
Over in `file_util.py` we have a `Autogenerated by Subiquity` blurb that is added to generated files, I think it would be good to include something similar.
https://github.com/canonical/subiquity/blob/main/subiquitycore/file_util.py#L75

Reply:
I teased out a new file_util.generate_timestamped_hearder` that can be used by generate_config_yaml and non_yaml file writes.

Reply:
Nice!